### PR TITLE
Updated 2 workshops

### PR DESCRIPTION
**Pull request by jvoight**
Screen Shot 2019-10-15 at 11 05 40 AM

### CODE REVIEW

**Comment by jwj61** [comment]
I thought the mod-ell modular forms took place in Warwick

**Comment by AndrewVSutherland** [comment]
@jwj61 Yes it did.

File: lmfdb/templates/workshops.html [hunk 1]
```diff
@@ -32,6 +32,20 @@
 
 <br>
 
+<li>
+Mod ell modular forms, <br>
+August 5-9, 2019,
+<a href="https://aimath.org/">American Institute of
```
Comment (JohnCremona) [inline lmfdb/templates/workshops.html]: Warwick Mathematics Institute https://warwick.ac.uk/fac/sci/maths/

**Comment by edgarcosta** [comment]
FYI, the test failures have nothing to do with this PR

File: lmfdb/templates/workshops.html (hunk 1 above)
Comment (jvoight) [reply lmfdb/templates/workshops.html]: Sorry, got confused by the AIM SQuaRE

**Pull request merged**